ci/build_libtelio.py

```diff
@@ -35,17 +35,19 @@
 
 
 def copy_bindings(config):
-    if "binding_src" in LIBTELIO_CONFIG[config.target_os]:
-        telio_bindings = f"{PROJECT_CONFIG.root_dir}/{LIBTELIO_CONFIG[config.target_os]['binding_src']}"
-        binding_destination = (
-            f"{PROJECT_CONFIG.root_dir}/{LIBTELIO_CONFIG[config.target_os]['binding_dest']}"
-            + telio_bindings.split("/")[-1]
-        )
-
-        if os.path.exists(binding_destination):
-            rutils.remove_tree_or_file(binding_destination)
-
-        rutils.copy_tree_or_file(telio_bindings, binding_destination)
+    # TODO(Mathias): reenable when packaging solution is done
```

Comment:
Do we have anything more for us to remind about these TODO's than just code comment? E.g. jira item under UniFFI epic or similar?